tooling/create-tauri-app/src/recipes/cljs.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019-2021 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+// import { join } from 'path'
+import { shell } from '../shell'
+import { Recipe } from '../types/recipe'
+
+const cljs: Recipe = {
+  descriptiveName: {
+    name: 'ClojureScript (https://clojurescript.org/)',
+    value: 'cljs'
+  },
+  shortName: 'cljs',
+  extraNpmDevDependencies: [],
+  extraNpmDependencies: [],
+  configUpdate: ({ cfg }) => ({
+    ...cfg,
+    distDir: `../public`,
+    devPath: 'http://localhost:3000',
+  }),
+  preInit: async ({ cwd, cfg, ci, packageManager }) => {
+    // create-cljs-app creates the folder for you
+    if (packageManager === 'yarn') {
+      await shell(
+        'yarn',
+        [
+          'create',
+          'cljs-app',
```

Comment:
I am not a coljurescript user but I quickly checked `create-cljs-app` and the [templates](https://clojurescript.org/guides/project-templates) mentioned on coljurescript official site and they all seem a bit outdated and not maintained. I think we should go with [`shadow-cljs`](https://github.com/thheller/shadow-cljs#quick-start) (`yarn create cljs-project`) instead, it is well maintained and comes with a sensible default features.

Reply:
Unfortunately, this project, [`create-cljs-project`](https://github.com/thheller/shadow-cljs/tree/master/packages/create-cljs-project) just doesn't have quite enough boilerplate to bootstrap a cljs-project. Here it is on [npmjs.com](https://www.npmjs.com/package/create-cljs-project).

Reply:
I see, it is fine then.